tests/Functional/ContentProjectRenderTest.php

```diff
@@ -177,6 +210,114 @@ public function testProjectTopGroupRender(): void {
     $file_download_link = $file_row->find('css', '.ecl-file__download');
     $this->assertContains('/test.pdf', $file_download_link->getAttribute('href'));
     $this->assertContains('Download', $file_download_link->getText());
+
+    // Assert bottom region - Stakeholders.
+    $project_stakeholder = $this->assertSession()->elementExists('css', 'div#project-stakeholder');
+
+    // Assert header.
+    $stakeholder_headers = $project_stakeholder->findAll('css', 'h2');
+    $this->assertEqual($stakeholder_headers[0]->getText(), 'Stakeholders');
+
+    // Assert Coordinators field.
+    $stakeholder_sub_headers = $project_stakeholder->findAll('css', 'h3');
+    $this->assertCount(1, $stakeholder_sub_headers);
+    $this->assertEqual($stakeholder_sub_headers[0]->getText(), 'Coordinators');
+    $this->assertOrganisation($project_stakeholder, 'coordinator');
+
+    // Unpublish Coordinator and publish Participant organisations.
+    $coordinator_organisation->set('status', 0);
+    $coordinator_organisation->save();
+    $participant_organisation->set('status', 1);
+    $participant_organisation->save();
+
+    // Reload the page.
+    $this->drupalGet($node->toUrl());
+
+    // Assert Participants field.
+    $project_stakeholder = $this->assertSession()->elementExists('css', 'div#project-stakeholder');
+    $stakeholder_sub_headers = $project_stakeholder->findAll('css', 'h3');
+    $this->assertCount(1, $stakeholder_sub_headers);
+    $this->assertEqual($stakeholder_sub_headers[0]->getText(), 'Participants');
+    $this->assertOrganisation($project_stakeholder, 'participant');
+  }
+
+  /**
+   * Creates organisation entity.
+   */
+  protected function getOrganisationEntity($name, $status) {
+    // Create image for logo.
+    $file = file_save_data(file_get_contents(drupal_get_path('theme', 'oe_theme') . '/tests/fixtures/placeholder.png'), "public://placeholder_$name.png");
+    $file->setPermanent();
+    $file->save();
+
+    $media = $this->container->get('entity_type.manager')->getStorage('media')->create([
+      'bundle' => 'image',
+      'name' => "Test image $name",
+      'oe_media_image' => [
+        'target_id' => (int) $file->id(),
+      ],
+      'uid' => 0,
+      'status' => 1,
+    ]);
+    $media->save();
+
+    $organisation = $this->organisationStorage->create([
```

Comment:
Use here the storage the same way as for media, and remove the class property.

Reply:
As it was discussed organisationStorage is a class property.